draft-mdt-quic-explicit-measurements.md

```diff
@@ -322,19 +340,20 @@ upstream paths with different latency characteristics.
 
 The use of the loss bits is negotiated using a transport parameter:
 
-loss_bits (0x1057):
+enable_network_troubleshooting (0x1057):
 
-: The loss bits transport parameter is an integer value, encoded as a
-  variable-length integer, that can be set to 0 or 1 indicating the level of
-  loss bits support.
+: The enable_network_troubleshooting transport parameter is a zero-length value.
 
-When loss_bits parameter is present, the peer is allowed to use reserved bits in
-the short packet header as loss bits if the peer sends loss_bits=1.
+The enable_network_troubleshooting parameter can be set by either endpoint.
+When enable_network_troubleshooting parameter is present, unilaterally or
+bilaterally, both endpoints MUST use reserved bits in the short packet header
+as loss bits.
 
-When loss_bits is set to 1, the sender will use reserved bits as loss bits if
-the peer includes the loss_bits transport parameter.
+When enable_network_troubleshooting parameter is present, endpoints SHOULD
+enable the QUIC latency Spin bit to improve the troubleshooting effectiveness.
```

Comment:
Do you mean here that the endpoint sending `enable_network_troubleshooting` parameter SHOULD (why not MUST?) enable QUIC latency spin bit?  Or you do mean that when `enable_network_troubleshooting` parameter is used by both endpoints then endpoints SHOULD/MUST enable QUIC latency spin bit?